lib/travis/build/addons/jwt.rb

```diff
@@ -0,0 +1,41 @@
+require 'travis/build/addons/base'
+require 'jwt'
+
+module Travis
+  module Build
+    class Addons
+      class Jwt < Base
+        SUPER_USER_SAFE = true
+
+        def before_before_script
+          secrets = nil
+          if config.is_a?(String)
+            secrets = [config]
+          else
+            secrets = config.values
```

Comment:
I think it is possible that `config` may not respond to `#values`. For example, when it is an array.